### PR TITLE
Expand on lamp message for Catalina users

### PR DESCRIPTION
**Pull request by rtimmons**
My personal laptop came back from the shop with Catalina. I tried `lamp` and hastily read the text and skipped the reboot. It didn't work. I cursed and googled. And read the SO article. And googled more. Then I finally read the text again. Rebooted and re-ran stuff and it worked. Let's help users avoid that.

### CODE REVIEW

File: src/lamplib/tasks/download.py [hunk 1]
```diff
@@ -49,27 +47,86 @@ def fetch_and_install(self):
...
-                        "sudo diskutil apfs addVolume disk1 APFSX Data -mountpoint /data \\\n"
-                        "sudo diskutil enableOwnership /data \\\n"
-                        "sudo chflags hidden /data \\\n"
-                        "echo \"LABEL=Data /data apfs rw\" | sudo tee -a /etc/fstab \\\n"
-                        "mkdir /data/mci"
-                        )
+                        f"""
+
+😲 You must create the parent directory {self._name} for the genny toolchain.
+   You are on On MacOS Catalina or later, so use use the synthetic.conf method.
```
Comment (guoyr) [inline src/lamplib/tasks/download.py:58]: 💅

File: src/lamplib/tasks/download.py [hunk 2]
```diff
@@ -49,27 +47,86 @@ def fetch_and_install(self):
...
+
+We wish we didn't have to do this.
+
+1️⃣  Step 1 1️⃣
+Run the followng command
+
+    echo 'data' | sudo tee -a /etc/synthetic.conf
```
Comment (guoyr) [inline src/lamplib/tasks/download.py:65]: I've just been using a symlink to an existing dir (https://apple.stackexchange.com/a/372500), which has been working for me on Catalina (and Big Sur). It seems a lot easier than simulating a volume 😄

Comment (guoyr) [reply src/lamplib/tasks/download.py:65]: oops didn't mean to approve.

Comment (rtimmons) [reply src/lamplib/tasks/download.py:65]: @guoyr per convo, this is the method I used and lets me build on catalina--seems the symlink method won't work after the build team's catalina updates.

Comment (guoyr) [reply src/lamplib/tasks/download.py:65]: Sweet. This is our best option short of a fully static genny build

**Comment by rtimmons** [comment]
evergreen merge

**Pull request merged**